src/pages/Graph/SummaryPanelNamespaceBox.tsx

```diff
@@ -119,19 +119,20 @@ export default class SummaryPanelNamespaceBox extends React.Component<
     const numSvc = boxed.filter(`node[nodeType = "${NodeType.SERVICE}"]`).size();
     const numWorkloads = boxed.filter(`node[nodeType = "${NodeType.WORKLOAD}"]`).size();
     const { numApps, numVersions } = this.countApps(boxed);
-    const numEdges = boxed.edges().size();
-    // when getting accumulated traffic rates don't count requests from injected service nodes
-    const nonServiceEdges = boxed.filter(`node[nodeType != "${NodeType.SERVICE}"][!isBox]`).edgesTo('*');
+    const numEdges = boxed.connectedEdges().size();
+    // incoming edges are from a different namespace or a different cluster
     const incomingEdges = namespaceBox
       .cy()
-      .nodes(`[${CyNode.namespace} != "${namespace}"], [${CyNode.cluster} != "${cluster}"], [?${CyNode.isRoot}]`)
+      .nodes(`[${CyNode.namespace} != "${namespace}"],[${CyNode.cluster} != "${cluster}"]`)
```

Comment:
I see you removed root nodes for incoming edges both in NS and Cluster boxes.
But this have the side effect that the incoming tab is not populated for gateways:

![image](https://user-images.githubusercontent.com/23639005/104505175-2431a780-55a9-11eb-811d-847ba956d1b5.png)

I though we wanted that, because the `SummaryPanelGraph` counts edges of root nodes as incoming traffic and it is more consistent with the "Total" tab. Anyway, I'm not sure if we want to count this for NS and Cluster boxes and I see it may be slightly confusing as the box may give the idea that traffic is not really "going in". So it's your choice.

Reply:
@israel-hdez I was going back and forth on this and so I asked @jmazzitelli for his opinion.  His initial reaction was to keep it as is, but then he too started to wonder whether it would be more consistent to include the boxed root nodes.  After some playing around we liked the idea of including the boxed root nodes (aka traffic source nodes) but also making them more obvious in the graph, so that users could more easily see the nodes contributing to incoming traffic.  So, we added a new badge.  We both liked it, and in chat you also were +1.  So, I have reverted things and so incoming traffic will include the boxed root nodes (as well as any edges entering the box), as well as be badged:

![image](https://user-images.githubusercontent.com/2104052/104605074-0c5f2f80-564c-11eb-82d4-81fc95e45b3c.png)